Sniff.php

```diff
@@ -477,6 +479,53 @@ public function inClassScope(PHP_CodeSniffer_File $phpcsFile, $stackPtr, $strict
 
 
     /**
+     * Verify whether a token is within a scoped use statement.
+     *
+     * PHPCS cross-version compatibility method.
+     *
+     * In PHPCS 1.x no conditions are set for a scoped use statement.
+     * This method works round that limitation.
+     *
+     * @param PHP_CodeSniffer_File $phpcsFile The file being scanned.
+     * @param int                  $stackPtr  The position of the token.
+     *
+     * @return bool True if within use scope, false otherwise.
+     */
+    public function inUseScope(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
+    {
+        static $isLowPHPCS, $ignoreTokens;
+
+        if (isset($isLowPHPCS) === false) {
```

Comment:
Only if we can - i.e. class variables.
In this case, they are static variables within a method, so visibility is not an option/does not apply.